apps/weshipit/pages/pricing.tsx

```diff
@@ -10,8 +10,28 @@ import {
 
 import { linksApi } from './api/links';
 import { Layout } from '../components/layout';
+import { getAllFaq } from './api/faq';
+import { RichTextField } from '@prismicio/client/*';
+import { PrismicRichText } from '@prismicio/react';
+import { useState } from 'react';
 
-export default function PricingPage() {
+interface FaqProps {
+  id: string;
+  data: {
+    question: RichTextField;
+    answer: RichTextField;
+  };
+}
+
+interface FaqSectionProps {
+  faq: FaqProps[];
+}
+
+export default function PricingPage({ faq }: FaqSectionProps) {
```

Comment:
please follow schema.org for SEO 

```
  /** @type {import('schema-dts').FAQPage} */
  const schema = {
    '@context': 'https://schema.org',
    '@type': 'FAQPage',
    mainEntity: faqs.data.questions.map((question) => ({
      '@type': 'Question',
      name: prismic.asText(question.question),
      acceptedAnswer: prismic.asHTML(question.answer),
    })),
  }
```

https://prismic.io/docs/schema-org-nextjs#rendering-faq-metadata